content/iOS/RSFormView/RSFormViewExample.md

```diff
@@ -1,47 +1,47 @@
 # Build an app with RSFormView in Swift
-*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team and recently has started to write about some of the challenges faced during iOS development. You can also follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
+*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team. He writes about the challenges faced in iOS development. You can follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
 
-Note: this post is meant to help you integrate RSFormView in your app, if you are interested in the motivation and implementation of RSFormView please take a look at the previous article [Link to previous article].
+Note: This blog helps you integrate RSFormView into your app. For more about the motivation and implementation of RSFormView see the previous article [Link to previous article].
 
-## Creating the project   
+### Create an Xcode project   
 
- During this article, we will be implementing a single view iOS application with RSFormView, step-by-step.
+ In this tutorial, we implement a single-view iOS application with RSFormView, step-by-step. To get started, create a new Xcode project and select Single View Application. 
+ Choose a name and a destination for the project, in this case, RSFormViewExample.
  
- Create a new Xcode project and select Single View Application. 
- Select a name and a destination for the project, in our case it'll be RSFormViewExample.
+ ### Install CocoaPods and RSFormView
 
-In order to integrate RSFormView we need Cocoapods, so we should install cocoapods and add a podfile in our project.
-You can find instructions on how to do this in cocoapods [guides](https://guides.cocoapods.org/using/using-cocoapods.html).
-Add this line in your podfile `pod RSFormView`, this will bring the last RSFormView available version whenever you run `pod install/update`.
-Now that you have RSFormView installed you should see a workspace in your project folder, open it. 
+  1. To integrate RSFormView, first, install Cocoapods and add a Podfile in your project folder.
+        Instructions can be found in the CocoaPods [guides](https://guides.cocoapods.org/using/using-cocoapods.html).
```

Comment:
Is this indentation intentional?

Reply:
No but if you look in the blob it has no effect on how the text is shown: https://github.com/rootstrap/blog/blob/review/copyWriterSuggestions/content/iOS/RSFormView/RSFormViewExample.md

Reply:
Awesome!